src/views/Profile.vue

```diff
@@ -166,6 +168,11 @@ export default class Profile extends Vue {
     return this.currentCollectionPage * this.first - this.first;
   }
 
+  get defaultNFTImage() {
+    const url = new URL(window.location.href);
+    return `${url.protocol}${url.hostname}/img/kodadot_logo_v1_transparent_400px.56bb186b.png`;
```

Comment:
`.56bb186b.` in file name is probably just bundle hash, you want to remove it :)